saltgui/static/scripts/api.js

```diff
@@ -6,8 +6,17 @@ class API {
     this._callMethod = this._callMethod.bind(this);
     this._fetch = this._fetch.bind(this);
     this._getRunParams = this._getRunParams.bind(this);
+    this._manualRunMenuSysDocPrepare = Documentation._manualRunMenuSysDocPrepare.bind(this);
+    this._manualRunMenuSysDocRun = Documentation._manualRunMenuSysDocRun.bind(this);
```

Comment:
Much much better to have this in a separate class.

Reply:
done
removed these 2 lines.
replaced with explicit parameter instead of binding to another class.
one of these functions did not even need `this`.